test/intl_suggestors/intl_importer_test.dart

```diff
@@ -371,6 +352,39 @@ void main() {
         );
       });
     });
+
+    test('when there are only package imports', () {
+      final imports = [
+        "import 'package:over_react/over_react.dart';",
+        "import 'package:react_material_ui/react_material_ui.dart';",
+      ];
+      final hasPackageImports = imports.any((importStatement) => importStatement.startsWith("import 'package:"));
+      final hasRelativeImports = imports.any((importStatement) => importStatement.startsWith("import '../"));
+      final importFormat = decideImportFormat(hasPackageImports, hasRelativeImports);
+      expect(importFormat, isTrue);
+    });
+
+    test('when there are only relative imports', () {
+      final imports = [
+        "import '../../../src/intl/datatables_intl.dart';",
+        "import '../lib/file.dart';",
+      ];
+      final hasPackageImports = imports.any((importStatement) => importStatement.startsWith("import 'package:"));
+      final hasRelativeImports = imports.any((importStatement) => importStatement.startsWith("import '../"));
+      final importFormat = decideImportFormat(hasPackageImports, hasRelativeImports);
+      expect(importFormat, isFalse);
+    });
+
+    test('when there are both package and relative imports', () {
+      final imports = [
+        "import 'package:over_react/over_react.dart';",
+        "import '../../../src/intl/datatables_intl.dart';",
+      ];
+      final hasPackageImports = imports.any((importStatement) => importStatement.startsWith("import 'package:"));
```

Comment:
This doesn't seem like the correct test. If we are using relative imports within the package (the only place we can) then we should be using a relative import for the _intl.dart file. Otherwise we might as well not bother doing this, because there are very few files that will only have imports from within their own package.

Reply:
The import format is "package" when package and relative imports exist.